pkgs/applications/misc/gramps/default.nix

```diff
@@ -1,16 +1,51 @@
-{ lib, fetchFromGitHub, gtk3, pythonPackages, intltool, gexiv2,
-  pango, gobject-introspection, wrapGAppsHook, gettext,
-# Optional packages:
- enableOSM ? true, osm-gps-map, glib-networking,
- enableGraphviz ? true, graphviz,
- enableGhostscript ? true, ghostscript
- }:
+{ lib
+, fetchFromGitHub
+, gtk3
+, pythonPackages
+, intltool
+, gexiv2
+, pango
+, gobject-introspection
+, wrapGAppsHook
+, gettext
+, glibcLocales
+, fetchpatch
+  # Optional packages:
+, enableOSM ? true
+, osm-gps-map
+, glib-networking
+, enableGraphviz ? true
+, graphviz
+, enableGhostscript ? true
+, ghostscript
+}:
```

Comment:
Is there a convention about this? `nix-fmt` does it the way it is before this pr.

Reply:
nixpkgs-fmt does it this way

Reply:
I guess it doesn't matter then.